packages/orbit-design-tokens/src/dictionary/definitions/component/heading/heading-font-size.json

```diff
@@ -28,19 +28,19 @@
       "title-3": {
         "font-size": {
           "type": "size",
-          "value": "{foundation.font-size.large}"
+          "value": "{foundation.font-size.extra-large}"
         }
       },
       "title-4": {
         "font-size": {
           "type": "size",
-          "value": "{foundation.font-size.normal}"
+          "value": "{foundation.font-size.large}"
```

Comment:
I have just noticed a problem. This is being ignored because in deprecated (line 92 of this file) the value defined is different. Therefore, when we do `yarn tokens build`, the value for the tokens is actually the deprecated one. I think this is not expected, and blocks the correctness of this token definition. Can you take a look at it?